docs/999-big-list-of-options.md

```diff
@@ -141,7 +141,7 @@ export default {
 /*
 const MyBundle = (function ($) {
   // code goes here
-}(window.jQuery));
+}(window.$));
```

Comment:
Nicely spotted. Actually, the generated code does not even contain a reference to "window" but this is fixed in the 1.0 branch

Reply:
Thanks, I didn't note that `umd(global.$) !== cjs + amd + iife($)`.